Tests/TuistDependenciesTests/SwiftPackageManager/SwiftPackageManagerInteractorTests.swift

```diff
@@ -1,28 +1,91 @@
+import TSCBasic
 import TuistCore
+import TuistGraph
 import TuistSupport
 import XCTest
 
 @testable import TuistDependencies
+@testable import TuistDependenciesTesting
 @testable import TuistSupportTesting
 
 final class SwiftPackageManagerInteractorTests: TuistUnitTestCase {
     private var subject: SwiftPackageManagerInteractor!
 
+    private var fileHandlerMock: MockFileHandler!
+
+    private var temporaryDirectoryPath: AbsolutePath!
+
     override func setUp() {
         super.setUp()
-        subject = SwiftPackageManagerInteractor()
+
+        do {
+            temporaryDirectoryPath = try TemporaryDirectory(removeTreeOnDeinit: true).path
```

Comment:
as mentioned above, you don't need to initialize this property
if that was not the case, you also wouldn't have to make `do ... catch` dance and just use `setUpWithError` function